ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/block/BlockManager.java

```diff
@@ -130,12 +166,13 @@ public SafeFuture<InternalValidationResult> validateAndImportBlock(
           InternalValidationResult.reject("Block (or its parent) previously marked as invalid"));
     }
 
-    SafeFuture<InternalValidationResult> validationResult = validator.validate(block);
+    final SafeFuture<InternalValidationResult> validationResult =
+        blockGossipValidator.validate(block);
```

Comment:
I'd pass this one through `blockBroadcastValidator`  too, so all production code is using it. It would be cleaner to my mind

Reply:
done